### PR TITLE
pwd, touch, mv, mkdir, rmdir, find, exa, eza, cat: add and update Korean translation

### PR DESCRIPTION
**Pull request by CodePsy-2001**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

---

**Note to other maintainers**: Our Korean reviewer (native speaker/code owner) has been inactive for a while, so I have validated all pages with Google translate, we can wait for a few more days for their review. If not reviewed by then, we can merge this PR without their review.

**Comment by IMHOJEONG** [comment]
It's been a while since I wrote a review. I'm really sorry for the delay due to personal reasons. Thanks again for your contribution. LGTM!

**Review by IMHOJEONG — APPROVED** [review]

**Comment by kbdharun** [comment]
> It's been a while since I wrote a review. I'm really sorry for the delay due to personal reasons. Thanks again for your contribution. LGTM!

That's fine, glad to have you back :)

**Pull request merged**
